.github/ISSUE_TEMPLATE/RELEASE.md

```diff
@@ -0,0 +1,26 @@
+# Update source data
+
+## Partner Agency Data
+
+> It's important to note that when pulling the most recent data from `db-knownprojects-data` repo that the latest commit of that repo's copy in `db-knownprojects` should match that of the data repo. Without double-checking, you could be pulling stale data. List of source data that should be updated (via `db-knownprojects-data`) but double checked in this repo (after updating the submodule):
```

Comment:
This is an issue template that we need to fundamentally fix - the naming of the partner agency files is a bit confusing

Reply:
this list seems to be a copy of the names in [the data repo issue template](https://github.com/NYCPlanning/db-knownprojects-data/blob/main/.github/ISSUE_TEMPLATE/DATA_UPDATE.md)

Reply:
@mbh329 see https://github.com/NYCPlanning/db-knownprojects/pull/366/commits/177b871a474ffac49b16ba85c43338694079440a for a first pass at filenames

could also use the relevant function names in `extractors.py` (like `esd_projects()` instead of `XXXX State Developments for Housing Pipeline.xlsx`)

Reply:
This is exactly what I was talking about, I made these changes in the https://github.com/NYCPlanning/db-knownprojects-data/pull/222/files as well

Reply:
As it was, we had gotten a file name that i supposed to match an older filename which was associated with an extractor function in the data repo, which became these sort of confusing names in the issue template

Reply:
so you prefer listing the function names in the readme? I can do that here to align with your PR in the data repo

Reply:
changed to use function names